### PR TITLE
Bug SMV-117 fixes & SMV-68 Changes

### PR DESCRIPTION
**Pull request by girishsawant999**

### CODE REVIEW

File: www/src/components/containers/login/components/otpInputScreen/index.tsx [hunk 1]
```diff
@@ -1,22 +1,57 @@
...
+    let _seconds = timer.minutes * 60 + timer.seconds - 1;
+    if (_seconds > 0) {
+      setTimeout(
+        () =>
+          setTimer({
+            minutes: parseInt(String(_seconds / 60)),
+            seconds: _seconds % 60
+          }),
+        1000
+      );
```
Comment (Pranav2612000) [inline www/src/components/containers/login/components/otpInputScreen/index.tsx:50]: Lets move the timer component and it's logic to a different component

Comment (girishsawant999) [reply www/src/components/containers/login/components/otpInputScreen/index.tsx]: Done

Comment (girishsawant999) [reply www/src/components/containers/login/components/otpInputScreen/index.tsx]: Done

**Comment by Pranav2612000** [comment]
Meged

**Pull request merged**